satellite/api.go

```diff
@@ -362,6 +362,15 @@ func NewAPI(log *zap.Logger, full *identity.FullIdentity, db DB,
 		)
 	}
 
+	{ // setup analytics service
```

Comment:
Any reason this was moved?

Reply:
there was a larger change this was a part of that involved passing the service into metainfo - it needed to be moved for that. For this change, it can be in either spot.